do-dont/doc/do-dont.md

```diff
@@ -28,6 +29,18 @@ import '@divriots/dockit-core/do-dont/dockit-do.define';
 import '@divriots/dockit-core/do-dont/define-all';
 ```
 
+## API
+
+```js story
+import manifest from '../../custom-elements.json';
+export const api = () =>
+  html`<api-docs
+    .manifest="${manifest}"
+    only="dockit-do,dockit-dont"
+    selected="dockit-do"
```

Comment:
`selected` should no be necessary and first one should be picked up automatically, but due to a bug (fixed in this PR https://github.com/open-wc/api-viewer-element/pull/155) it can't be yet, waiting for the PR to be released